test

PeterThramkrongart opened the pull request.

Commit:
committing assignment files

PeterThramkrongart commented:
rest

PeterThramkrongart committed:
Merge pull request #1 from CDS-AU-DK/main

test

adivea commented:
Hi Peter, are you trying to contribute improved assignment scripts or just pushing to the wrong repo?

PeterThramkrongart commented:
<!--
/* Font Definitions */
@font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;}
/* Style Definitions */
p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin:0in;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
a:link, span.MsoHyperlink
	{mso-style-priority:99;
	color:blue;
	text-decoration:underline;}
.MsoChpDefault
	{mso-style-type:export-only;}
@page WordSection1
	{size:8.5in 11.0in;
	margin:85.05pt 56.7pt 85.05pt 56.7pt;}
div.WordSection1
	{page:WordSection1;}
-->I was to gran you  changes to repo , but I had the merge order the wrong way around…  From: Adela SobotkovaSent: Wednesday, February 17, 2021 08:31To: CDS-AU-DK/cds-spatialCc: Peter Thramkrongart; AuthorSubject: Re: [CDS-AU-DK/cds-spatial] test (#2) Hi Peter, are you trying to contribute improved assignment scripts or just pushing to the wrong repo?—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe. 

PeterThramkrongart committed:
Merge pull request #2 from CDS-AU-DK/main

test

PeterThramkrongart committed:
Merge pull request #3 from CDS-AU-DK/main

update repo

Commit:
test

Commit:
test

Merge branch 'main' of https://github.com/PeterThramkrongart/cds-spatial into main

# Conflicts:
#	Week04/SA11_RasterAnalysis_exercise.Rmd

Commit:
test

Commit:
test commit

PeterThramkrongart committed:
Merge pull request #4 from CDS-AU-DK/main

W06 exercises

Commit:
added Week06

Commit:
updating repo

Commit:
update repo

Commit:
update repo
